MarkdownAttributedString.podspec

```diff
@@ -0,0 +1,16 @@
+Pod::Spec.new do |s|
+  s.name = 'MarkdownAttributedString'
+  s.version = '0.1'
+  s.license = 'MIT'
+  s.author = 'Craig Hockenberry'
+  s.summary = 'Adding Markdown support to NSAttributedString'
+  s.homepage = 'https://github.com/chockenberry/MarkdownAttributedString'
+  s.source = { :git => 'https://github.com/chockenberry/FetchRequests.git', :tag => s.version }
```

Comment:
s/FetchRequests/MarkdownAttributedString

Also, why not go straight for 1.0.0? :)

Reply:
Whoops! Fixed
And, that felt like a decision for Craig to make